core/src/display_object/edit_text.rs

```diff
@@ -73,39 +73,12 @@ pub struct EditTextData<'gc> {
     /// rendering.
     text_spans: FormatSpans,
 
-    /// If the text is in multi-line mode or single-line mode.
-    is_multiline: bool,
-
-    /// If the text can be selected by the user.
-    is_selectable: bool,
-
-    /// If the text can be edited by the user.
-    is_editable: bool,
-
-    /// If the text is word-wrapped.
-    is_word_wrap: bool,
-
-    /// If this is a password input field
-    is_password: bool,
-
-    /// If the text field should have a background. Only applied when has_border.
-    has_background: bool,
-
     /// The color of the background fill. Only applied when has_border and has_background.
     background_color: u32,
```

Comment:
I think `background_color` and `border_color` should be `swf::Color`, but that's for a future PR.